recipes-browser/cog/cog-meson.inc

```diff
@@ -4,7 +4,7 @@ PACKAGECONFIG[dbus] = "-Dcog_dbus_control=system -Dcog_dbus_system_owner=${COG_D
 PACKAGECONFIG[drm] = ",,wpebackend-fdo libdrm virtual/libgbm libinput"
 PACKAGECONFIG[gtk4] = ",,gtk4"
 PACKAGECONFIG[headless] = ",,wpebackend-fdo"
-PACKAGECONFIG[soup2] = "-Dsoup2=enabled,-Dsoup2=disabled,libsoup-2.4"
+PACKAGECONFIG[soup2] = "-Dwpe_api=1.0,,libsoup-2.4"
```

Comment:
Isn't there `libsoup3` yet in Yocto? I would like to encourage switching to it if possible 😺

Reply:
This is branch is still offering support to the dunfell and other intermediate Yocto release. libsoup 3 will be available from kirkstone and so. This is being addressed in https://github.com/Igalia/meta-webkit/pull/489